src/main/java/com/amazon/opendistroforelasticsearch/security/transport/OpenDistroSecurityRequestHandler.java

```diff
@@ -202,10 +209,14 @@ protected void messageReceivedDecorate(final T request, final TransportRequestHa
                         || HeaderHelper.isTrustedClusterRequest(getThreadContext())) {
 
                     final String userHeader = getThreadContext().getHeader(ConfigConstants.OPENDISTRO_SECURITY_USER_HEADER);
+                    final String injectedUserHeader = getThreadContext().getHeader(ConfigConstants.OPENDISTRO_SECURITY_INJECTED_USER_HEADER);
 
                     if(Strings.isNullOrEmpty(userHeader)) {
                         //user can be null when a node client wants connect
                         //getThreadContext().putTransient(ConfigConstants.OPENDISTRO_SECURITY_USER, User.OPENDISTRO_SECURITY_INTERNAL);
```

Comment:
Should this be removed if not needed?

Reply:
Seems to be from past PR. Removed either way.